src/middlewared/middlewared/plugins/alert.py

```diff
@@ -399,6 +400,9 @@ def restore(self, id):
                             pass
                         else:
                             raise
+                except ReserveFDException:
```

Comment:
How are we going to know the alert failed to run on remote side if we just pass here?

Reply:
We won't, that's why I had a debug log message. I'd like to add that back at the caller side if you agree.

Reply:
Definitely is a strong word. It could be a one-off.

Why you dont want to raise on hardware alerts? So a support ticket is not created? I dont see a harm in only logging it. Whats the benefit of raising?

Reply:
> Whats the benefit of raising?
We'll get "running alert on passive node failed due to ReserveFDException" for all non-hardware alert sources if we raise. Otherwise it would only be a line in log file.

Reply:
Support is fine with just a log entry.

Reply:
Yeah, not sure its any useful to raise an alert with that error?

Reply:
@william-gr I don't believe it is. So do you want me to push PR to log message in alert plugin?

Reply:
Please do, unless @themylogin feels strongly about the alert instead? ; )

Reply:
If support is fine with log, I am also fine :)